server/src/server.ts

```diff
@@ -35,7 +35,7 @@ import { addItemAmount, getItemById, getItems, getItemsWhereStaff, getItemsWhere
 import { InventoryItem } from "./schemas/storeFrontSchema.js";
 import { createLedger } from "./repositories/Store/InventoryLedgerRepository.js";
 import { getZoneHoursNextWeek } from "./repositories/Zones/ZoneHoursRepository.js";
-import * as Atrium from "./integrations/atrium-integration/atrium.js"
```

Comment:
my unused import 
![an-emoji-disintegrating-in-pain](https://github.com/user-attachments/assets/14ce296d-133e-4dce-af55-a62f3ea73aef)